### PR TITLE
replace all instances of $.timeago with moment.fromNow

### PR DESCRIPTION
**Pull request by Hamms**
So the result can be internationalized

Screenshots
| --- |
image |
image |
image

Note that although ProjectCard is using the new component, it won't be internationalized since the `/projects` page doesn't load pageConstants (or provide any other way of determining the locale)

### CODE REVIEW

**Review by joshlory — APPROVED** [review]
Awesome! Looks like there are only a few remaining usages of `$.timeago` before we can remove that vendor script entirely.

**Review by Erin007 — APPROVED** [review]
LGTM! Should we be thinking about internationalizability for other places where we use times?  For example, the teacher dashboard assessment submission status table we have a timestamp that doesn't use timeago, but I think the month and day would flip if we were fully international-friendly. Super small, but thought I'd note it somewhere!
Screen Shot 2019-05-21 at 1 35 55 PM

**Comment by Hamms** [comment]
We definitely should! Would you add a Jira card in the foundations project to track that?

**Comment by Erin007** [comment]
> We definitely should! Would you add a Jira card in the foundations project to track that?

will do! https://codedotorg.atlassian.net/browse/FND-483

**Comment by Hamms** [comment]
> Awesome! Looks like there are only a few remaining usages of $.timeago before we can remove that vendor script entirely.

@joshlory actually, I think I got em all! I'll open a new PR removing it

**Comment by Hamms** [comment]
@Erin007 @joshlory PTAL; I made a bunch of changes to tests to get them to pass

File: apps/src/templates/TimeAgo.jsx [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+      </time>
+    );
+  }
+}
+
+// expose the "unconnected" version of the component as specifically
+// "unlocalized", to support usage in places where we don't have access to the
+// redux store or the locale and to make it clear what we lose in that
+// situation
+export const UnlocalizedTimeAgo = TimeAgo;
```
Comment (Erin007) [inline apps/src/templates/TimeAgo.jsx:33]: super teeny nit: `// Use capitalization and punctuation in comments.`

**Review by Erin007 — APPROVED** [review]
looks good!

**Pull request merged**